packages/lesswrong/components/votes/SmallSideVote.tsx

```diff
@@ -9,14 +9,18 @@ const SmallSideVote = ({document, hideKarma=false, collection}: {
 }) => {
   const votingSystemName = (document as any)?.votingSystem || "default";
   const votingSystem = getVotingSystemByName(votingSystemName);
-  
-  const VotingComponent = votingSystem.getCommentVotingComponent();
-  return <VotingComponent
-    document={document}
-    hideKarma={hideKarma}
-    collection={collection}
-    votingSystem={votingSystem}
-  />
+
+  const VotingComponent = votingSystem.getCommentVotingComponent?.();
```

Comment:
Under what circumstances will this be null?

Reply:
If somebody tried to use the election candidate voting system for a post (which is possible in theory, but not given as an option by the UI).